src/main/webapp/jsp/fp/ProcurementCard.jsp

```diff
@@ -0,0 +1,71 @@
+<%--
```

Comment:
I think it would be better to define a new jsp for the CBCP doc. We want to avoid customizing base code. It will also be confusing down the road to maintain this - one would expect a CorporateBilledCorporatePaid.jsp for CBCP instead of a ProcurementCard.jsp